### DISCUSSION TITLE
Welcome to AutoBeam Discussions!

### DISCUSSION THREAD

**Discussion by peskyboyz** [peskyboyz/AutoBeamAnimationGenerator#1] [Announcements]
If you have any feedback regarding the AutoBeam tool, please leave it here.

**Comment by CoraBBX** [comment]
ok, first comment on this here. But I have a good question about the boost gauge system. Would it be possible to add more options sense Twincharged and Compound turbo have been added? And maybe Sequential as well sense they have different levels of boost as well. I don't know how automation exports boost, but it would be cool if that was possible for a future update.

**Reply by peskyboyz** [reply]
> ok, first comment on this here. But I have a good question about the boost gauge system. Would it be possible to add more options sense Twincharged and Compound turbo have been added? And maybe Sequential as well sense they have different levels of boost as well. I don't know how automation exports boost, but it would be cool if that was possible for a future update.

Hello! Unfortunately, due to the way that the animations in BeamNG work this isn't possible. What happens is that BeamNG has a large set of variables (called electrics), like 'RPM', 'steering', 'wheelspeed', etc., that can be accessed to make animations. This is the way that vanilla BeamNG cars work as well. Currently Beam only has 'boost' and 'superchargerboost' (which will be available in the next AutoBeam update), and these two are the only ways to access the air intake pressure of the engines. So unless the Beam devs add a new car with twin charging and create a new boost electric, this won't ever be possible.

**Reply by CoraBBX** [reply]
ok, well the beam community should start asking for that. But when you update AutoBeam, I will try the 2 boost systems out and see if they act different when used on a twincharged car. I will be sure to report on what happens so you can update the help guide.

Here is what I was making when I though of this though
<img width="942" height="963" alt="Screenshot 2025-12-13 124837" src="https://github.com/user-attachments/assets/de76c51e-f74e-47ee-8c2b-e5c4887e8390" />
<img width="1974" height="1081" alt="Screenshot 2025-12-13 124858" src="https://github.com/user-attachments/assets/eeb4c29b-7ed6-460d-9295-adbac80b7a3c" />
<img width="2559" height="1365" alt="Screenshot 2025-12-13 125107" src="https://github.com/user-attachments/assets/d5bae20d-463a-4ca9-b568-6ebc5740fe36" />
<img width="1674" height="815" alt="Screenshot 2025-12-13 164405" src="https://github.com/user-attachments/assets/331e5d30-3b2d-4eb5-ad38-3d2c3444fc1c" />
<img width="787" height="617" alt="Screenshot 2025-12-13 164804" src="https://github.com/user-attachments/assets/c24bac2c-e12a-427b-9375-39dc1e97aa92" />

**Reply by CoraBBX** [reply]
And just relized with the notepad of code. I was going try to do a changing oil and coolant temp lights. Would doing it like the rpm light coding the min and max 1 apart work the same?
